contrib/etc/mariadb_environment

```diff
@@ -29,17 +27,21 @@ if [ -z "${MEMORY_LIMIT_IN_BYTES:-}" ]; then
     innodb_buffer_pool_size='32M'
     innodb_log_file_size='8M'
     innodb_log_buffer_size='8M'
+    max_connections='16'
 else
-    key_buffer_size="$(python -c "print(int((${MEMORY_LIMIT_IN_BYTES}/(1024*1024))*0.1))")M"
-    read_buffer_size="$(python -c "print(int((${MEMORY_LIMIT_IN_BYTES}/(1024*1024))*0.05))")M"
-    innodb_buffer_pool_size="$(python -c "print(int((${MEMORY_LIMIT_IN_BYTES}/(1024*1024))*0.5))")M"
-    innodb_log_file_size="$(python -c "print(int((${MEMORY_LIMIT_IN_BYTES}/(1024*1024))*0.15))")M"
-    innodb_log_buffer_size="$(python -c "print(int((${MEMORY_LIMIT_IN_BYTES}/(1024*1024))*0.15))")M"
+    # memory section
+    key_buffer_size=$(( $MEMORY_LIMIT * 8 / 100 ))M
+    read_buffer_size=$(( $MEMORY_LIMIT * 4 / 100 ))M
+    innodb_buffer_pool_size=$(( $MEMORY_LIMIT * 44 / 100 ))M
+    innodb_log_file_size=$(( $MEMORY_LIMIT * 12 / 100 ))M
+    innodb_log_buffer_size=$(( $MEMORY_LIMIT * 12 / 100 ))M
+    #/ memory section
+    max_connections=$(( $MEMORY_LIMIT / 512 ))
```

Comment:
This should be 16 connections per 512

You then need a second line like:
`[[ $max_connections -le 16 ]] && max_connections=16`